plugins/twitter/classes/DataProvider/Twitter.php

```diff
@@ -109,7 +109,7 @@ public function fetch($limit = FALSE) {
 				$additional_data = [];
 
 				// Skip retweets
-				if ($status['retweeted_status']) {
+				if (array_key_exists('retweeted_status', $status) && array_key_exists('text', $status['retweeted_status'])) {
```

Comment:
@tuxpiper is this basically `!empty($status['retweeted_status']`? or is it specifically the `text` key that matters?